cob_bms_driver/src/fake_bms.py

```diff
@@ -25,66 +25,66 @@
 
 class FakeBMS(object):
     def __init__(self):
-        self.srv_current              = rospy.Service('~set_current', SetFloat, self.current_cb)
-        self.srv_relative_remaining_capacity   = rospy.Service('~set_relative_remaining_capacity', SetInt, self.relative_remaining_capacity_cb)
-        self.poll_frequency           = rospy.get_param('~poll_frequency_hz', 20.0)
-        self.pub_voltage              = rospy.Publisher('~voltage', Float64, queue_size = 1)
-        self.pub_current              = rospy.Publisher('~current', Float64, queue_size = 1)
-        self.pub_remaining_capacity   = rospy.Publisher('~remaining_capacity', Float64, queue_size = 1)
-        self.pub_full_charge_capacity = rospy.Publisher('~full_charge_capacity', Float64, queue_size = 1)
-        self.pub_temparature          = rospy.Publisher('~temperature', Float64, queue_size = 1)
+        self._srv_current              = rospy.Service('~set_current', SetFloat, self.current_cb)
+        self._srv_relative_remaining_capacity   = rospy.Service('~set_relative_remaining_capacity', SetInt, self.relative_remaining_capacity_cb)
+        self._poll_frequency           = rospy.get_param('~poll_frequency_hz', 20.0)
+        self._pub_voltage              = rospy.Publisher('~voltage', Float64, queue_size = 1)
+        self._pub_current              = rospy.Publisher('~current', Float64, queue_size = 1)
+        self._pub_remaining_capacity   = rospy.Publisher('~remaining_capacity', Float64, queue_size = 1)
+        self._pub_full_charge_capacity = rospy.Publisher('~full_charge_capacity', Float64, queue_size = 1)
+        self._pub_temparature          = rospy.Publisher('~temperature', Float64, queue_size = 1)
 
-        self.updater = Updater()
-        self.updater.setHardwareID("bms")
-        self.updater.add("cob_bms_dagnostics_updater", self.produce_diagnostics)
+        self._updater = Updater()
+        self._updater.setHardwareID("bms")
+        self._updater.add("cob_bms_dagnostics_updater", self.produce_diagnostics)
 
-        self.voltage              = 48.0
-        self.current              = -8.0
-        self.remaining_capacity   = 35.0
-        self.full_charge_capacity = 35.0 # Ah
-        self.temperature          = 25.0
+        self._voltage              = 48.0 # V
+        self._current              = -6.0 # A
+        self._remaining_capacity   = 60.0 # Ah
+        self._full_charge_capacity = 60.0 # Ah
+        self._temperature          = 25.0 # °C
```

Comment:
I would make those configurable via a new `battery.yaml` in `mojin_hardware_config`, i.e. `rospy.get_param('voltage', 48)` defaulting to the old/cob4 values...i.e. open a PR that adds these `battery.yaml` file in `mojin_robots` and add a `rosparam` line in the respective `mojin_bringup` launch files

Reply:
make sure the `battery.yaml` includes also the values for https://github.com/ipa320/cob_docking/pull/84#discussion_r941630133

Reply:
done in https://github.com/mojin-robotics/mojin_robots/pull/515